### PR TITLE
AFC-1586 - res_interpreter_language: interpreter languages

### PR DESCRIPTION
**Pull request by nils-vertel**
Multi repo pull request:
- odoo-af
- odoo-base
- odoo-outplacement
- odoo-project

### CODE REVIEW

File: res_interpreter_language/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import models  # noqa: F401
```
Comment (arturchilangwa-vertel) [inline res_interpreter_language/__init__.py:1]: what does the comment stand for?

File: res_interpreter_language/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import res_interpreter_language  # noqa:F401
```
Comment (arturchilangwa-vertel) [inline res_interpreter_language/models/__init__.py:1]: Same here

File: res_interpreter_language/views/res_interpreter_language_view.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    <data>
+        <record id="view_interpreter_language_form" model="ir.ui.view">
+            <field name="name">res.interpreter_language.form</field>
+            <field name="model">res.interpreter.language</field>
+            <field name="arch" type="xml">
+                <form string="Interpreter Language">
+                    <sheet>
+                        <!--<h1>
+                            <field name="name" default_focus="1" placeholder="Name" />
+                        </h1> -->
```
Comment (arturchilangwa-vertel) [inline res_interpreter_language/views/res_interpreter_language_view.xml]: Please state in a code comment above why you want to keep this code

File: res_interpreter_language/views/res_interpreter_language_view.xml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            </field>
+        </record>
+        <record id="action_interpreter_language_form" model="ir.actions.act_window">
+            <field name="name">Interpreter Language</field>
+            <field name="type">ir.actions.act_window</field>
+            <field name="res_model">res.interpreter.language</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">tree,form</field>
+            <field name="view_id" ref="view_interpreter_language_tree"/>
+            <field name="context">{}</field>
```
Comment (arturchilangwa-vertel) [inline res_interpreter_language/views/res_interpreter_language_view.xml]: Do we need the `context` field if it only passes in a empty dict?

**Review by arturchilangwa-vertel — CHANGES_REQUESTED** [review]

File: res_interpreter_language/__init__.py (hunk 1 above)
Comment (nils-vertel) [reply res_interpreter_language/__init__.py:1]: It tells the Linter to ignore the Fault F401 (Imported function is not used)

File: res_interpreter_language/models/__init__.py (hunk 2 above)
Comment (nils-vertel) [reply res_interpreter_language/models/__init__.py:1]: It tells the Linter to ignore the Fault F401 (Imported function is not used)

File: res_interpreter_language/views/res_interpreter_language_view.xml (hunk 3 above)
Comment (nils-vertel) [reply res_interpreter_language/views/res_interpreter_language_view.xml]: I don't want to keep it, I'll remove.

File: res_interpreter_language/views/res_interpreter_language_view.xml (hunk 4 above)
Comment (nils-vertel) [reply res_interpreter_language/views/res_interpreter_language_view.xml]: Removing

**Pull request merged**